Targets/QTJS/README.md

```diff
@@ -0,0 +1,11 @@
+# Target: Qt QJSEngine
+
+
+To build the Qt QJSEngine for fuzzing:
+
+1. clone the repo: `git clone git://code.qt.io/qt/qt5.git qt6`
```

Comment:
Nit: qt5 or qt6?

Reply:
So the qt5 repository shares the code for qt5 and qt6 within different branches. The hash in REVISION would be used to checkout qt6.3.1

Reply:
Ah ok, maybe just add something like ` (the qt5 repo contains the code for both qt5 and qt6 in different branches)` at the end of the line or so?

Reply:
done :)